### PR TITLE
My experience at smashing conference

### PR DESCRIPTION
**Pull request by jackdcrawford**
### Blog title
My smashing experience

### Blog description
A summary of the talks that I attended at Smashing Conference Oxford 2015

#### Author
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 1
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 2
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

### CODE REVIEW

**Comment by jodiedoubleday** [comment]
Taking Editor 1

**Comment by jodiedoubleday** [comment]
A mind dump of all things smashing conf :) good stuff :+1:

**Comment by Joezo** [comment]
Taking editor 2

File: _posts/2015-04-10-my-smashing-experience.md [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+---
+
+## Don't give them what they want, give them what they need - Richard Rutter [@clagnut](http://twitter.com/clagnut)
+_A case study on designing the [The Royal Borough of Kensington and Chelsea](http://www.rbkc.gov.uk/) website_
+
+* Identify what the client actually _needs_, not what they _want_. This is essential but is usually hard!
+* Don't forget the call centre! They are a valuable user research resource that shouldn't be ignored
+* All pages should have one primary goal split things that do two things into two pages
```
Comment (Joezo) [inline _posts/2015-04-10-my-smashing-experience.md]: feels like it needs a comma after `primary goal`

File: _posts/2015-04-10-my-smashing-experience.md [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+* Use colour intensity to represent magnitude
+* [colorbrewer2.org](http://colorbrewer2.org)
+
+<blockquote class="twitter-tweet" lang="en"><p>In data visualisation, color intensity drives to investigate. Color hue drives to alarm (red) <a href="https://twitter.com/natalieyadren">@natalieyadren</a> <a href="https://twitter.com/hashtag/smashingconf?src=hash">#smashingconf</a></p>&mdash; Hungry Geek (@TheHungryGeek) <a href="https://twitter.com/TheHungryGeek/status/577867329620566016">March 17, 2015</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+---
+
+## Building beyond the browser - Tom Gianattasio [@attasi](http://twitter.com/attasi)
+_How the guys at [macaw](http://macaw.co/) used [node-webit](http://nwjs.io/) to build their desktop hybrid app_
```
Comment (Joezo) [inline _posts/2015-04-10-my-smashing-experience.md]: `node-webit`

File: _posts/2015-04-10-my-smashing-experience.md [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+* €80000/week taken via social media payments at [KLM](https://www.klm.com/)
+* [Awesome dog advert](https://www.youtube.com/watch?v=NK-T_t166TY)
+
+<blockquote class="twitter-tweet" lang="en"><p><a href="https://twitter.com/hashtag/smashingconf?src=hash">#smashingconf</a> – “Crafting for <a href="https://twitter.com/hashtag/worlddomination?src=hash">#worlddomination</a>” by <a href="https://twitter.com/Polledemaagt">@polledemaagt</a>. <a href="http://t.co/eoCqGuAP4b">pic.twitter.com/eoCqGuAP4b</a></p>&mdash; Elisabeth Irgens (@elisabethirg) <a href="https://twitter.com/elisabethirg/status/578214540317483008">March 18, 2015</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+---
+
+## User research for designers & engineers - Rachel Ilan Simpson [@rilan](http://twitter.com/rilan)
+_User research is _really_ scary but absolutely essential_
```
Comment (Joezo) [inline _posts/2015-04-10-my-smashing-experience.md]: not sure what's going with the `_`s here, it looks a bit odd

**Comment by Joezo** [comment]
Couple of typos to sort out, looks good other than that though :)

**Comment by Joezo** [comment]
:+1: :fire_engine:

**Pull request merged**